docs/examples/3d_n_body_pendulum.rst

```diff
@@ -0,0 +1,492 @@
+
+===================
+3d_n_body_pendulum
+===================
+.. note::
+
+   You can download this example as a Python script:
+   :jupyter-download:script:`3d_n_body_pendulum` or Jupyter notebook:
+   :jupyter-download:notebook:`3d_n_body_pendulum`.
+
+.. jupyter-execute::
+    import sympy as sm
```

Comment:
The error I'm seeing is this:
```
/home/runner/work/pydy/pydy/docs/examples/3d_n_body_pendulum.rst.rst:11: WARNING: Ignoring inline code in Jupyter cell included from "examples/import sympy as sm
```

I think that you need a blank line after every `.. jupyter-execute::` line. One blank line before the code starts.